doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -386,22 +387,16 @@ It is recommended to follow the “url” link values,
 applicable to retrieve resources. Instead of constructing your own URLs,
 to keep your client decoupled from implementation details.
 
-+ Attributes(object)
-    + entities_url (required, string) - URL which points to the entities resource
-    + types_url (required, string) - URL which points to the types resource
-    + subscriptions_url (required, string) - URL which points to the subscriptions resource
-    + registrations_url (required, string) - URL which points to the registrations resource
 
 + Response 200 (application/json)
 
-        {
```

Comment:
As far as I have checked, this is the only piece of JSON that you have removed from .apib in this PR. Is that correct?